addons/mail/static/tests/crosstab/crosstab_tests.js

```diff
@@ -114,15 +115,17 @@ QUnit.test("Channel subscription is renewed when channel is left", async (assert
     const pyEnv = await startServer();
     pyEnv["discuss.channel"].create({ name: "Sales" });
     const { env, openDiscuss } = await start();
+    const updateChannelDeferred = makeExpirableDeferred();
     patchWithCleanup(env.services["bus_service"], {
         forceUpdateChannels() {
             assert.step("update-channels");
+            updateChannelDeferred.resolve();
         },
     });
     openDiscuss();
     await click(".o-mail-DiscussSidebarChannel .btn[title='Leave this channel']");
     await contains(".o-mail-DiscussSidebarChannel", { count: 0 });
-    await new Promise((resolve) => setTimeout(resolve)); // update of channels is debounced
+    await updateChannelDeferred;
```

Comment:
It's not possible to use `waitForChannels` or `waitUntilSubscribe` or something like that?

Reply:
updated to use `waitUntilSubscribe`